packages/cli/src/config/settings.ts

```diff
@@ -64,7 +64,6 @@ const MIGRATION_MAP: Record<string, string> = {
   autoAccept: 'tools.autoAccept',
   autoConfigureMaxOldSpaceSize: 'advanced.autoConfigureMemory',
   bugCommand: 'advanced.bugCommand',
-  chatCompression: 'model.chatCompression',
```

Comment:
Should we actually be migrating old chat compression bits to the new one? Why remove? Just throwing this comment out for introspection, don't feel like you need to respond to it though